apps/api/src/test-data/utilities/convertInMemorySnapshotToDatabaseFormat.ts

```diff
@@ -0,0 +1,57 @@
+import BaseDomainModel from '../../domain/models/BaseDomainModel';
+import { AggregateId } from '../../domain/types/AggregateId';
+import {
+    InMemorySnapshot,
+    InMemorySnapshotOfResources,
+    ResourceType,
+} from '../../domain/types/ResourceType';
+import { UuidDocument } from '../../lib/id-generation/types/UuidDocument';
+import { ArangoCollectionId } from '../../persistence/database/collection-references/ArangoCollectionId';
+import { ArangoEdgeCollectionId } from '../../persistence/database/collection-references/ArangoEdgeCollectionId';
+import { getArangoCollectionIDFromResourceType } from '../../persistence/database/collection-references/getArangoCollectionIDFromResourceType';
+import buildEdgeDocumentsFromCategoryNodeDTOs from '../../persistence/database/utilities/category/buildEdgeDocumentsFromCategoryNodeDTOs';
+import mapCategoryDTOToArangoDocument from '../../persistence/database/utilities/category/mapCategoryDTOToArangoDocument';
+import mapEdgeConnectionDTOToArangoEdgeDocument from '../../persistence/database/utilities/mapEdgeConnectionDTOToArangoEdgeDocument';
+import mapEntityDTOToDatabaseDTO from '../../persistence/database/utilities/mapEntityDTOToDatabaseDTO';
+import { DTO } from '../../types/DTO';
+
+type InMemoryDatabaseSnapshot = {
+    documentCollections: {
+        [K in Exclude<ArangoCollectionId, 'uuids'>]: unknown[];
```

Comment:
Interesting, I'll have to look up the difference between Exclude and Omit...

Reply:
I'll explain in Slack.